src/app/(post-login)/submission/components/Search.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+'use client';
+
+import { Input, MenuItem, css } from '@icgc-argo/uikit';
+
+export default function Search({
+	query,
+	onChange,
+}: {
+	query: string;
+	onChange: (s: string) => void;
+}) {
+	return (
+		<MenuItem
+			level={1}
+			selected
+			contentAs="div"
+			content={
+				<Input
+					aria-label="programs search"
+					onChange={(e) => {
+						onChange(e.target.value);
```

Comment:
not a telegram 😆 we don't charge by the letter
```suggestion
					onChange={(event) => {
						onChange(event.target.value);
```

Reply:
https://github.com/icgc-argo/rdpc-ui/pull/48/commits/3cd6fb696c7ab2f4c36f3ec95704397344bb6f53